manifests/gcp_marketplace/schema.yaml

```diff
@@ -175,24 +175,20 @@ properties:
     default: false
     description: |-
       Kubeflow Pipelines default to use docker executor. Select this option to
-      use [the emissary executor (Alpha)](https://github.com/kubeflow/pipelines/issues/6249).
+      use the emissary executor.
 
-      Refer to migrating to emissary executor documentation for migration instructions.
+      Warning, docker executor does not work on Google Kubernetes Engine (GKE) 1.19+
+      by default, because docker container runtime is no longer the default. Furthermore,
+      docker container runtime will be deprecated on GKE 1.20+.
 
-      Emissary executor is more portable. On Google Kubernetes Engine (GKE)
-      clusters 1.19+, containerd is the default container runtime instead of
-      docker, so docker executor does not work properly. Emissary executor is not
-      coupled to a specific container runtime, so it works properly on any
-      Kubernetes clusters including GKE 1.19+ with containerd as runtime.
+      To prepare for the migration to emissary executor, refer to [1] for more details.
```

Comment:
".....refer to [1] for more details." -> "refer to [Argo Workflow Executors][1] for more details."